### DISCUSSION TITLE
GKE support

### DISCUSSION THREAD

**Discussion by weisinc** [prowler-cloud/prowler#3292] [Q&A] [answered]
GKE is not in the list of services for GCP scans. Anyone know if there are plans to support it, and if so when?

**Comment by toniblyx** [comment]
Hello! We are working on the Kubernetes provider right now, that will help on the support for GKE too. We plan to release support for Kubernetes covering the whole CIS benchmark for Prowler v4 coming up by March 2024.

**Reply by localh0rs7** [reply]
hi,
is there any Update on this matter. I would like to see this feature too, since I have problems with assessing GCP CIS with other tools, that already support the CIS GKE.

**Reply by pedrooot** [reply, marked as answer]
Hey! @localh0rs7 you can use prowler for the GKE service adding `--service gke` to your Prowler execution command for GCP. To make a scan of your Kubernetes infra use `prowler kubernetes`, more [docs here](https://docs.prowler.com/projects/prowler-open-source/en/latest/tutorials/kubernetes/in-cluster/).
Thanks for using Prowler! 🚀
